delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/resolvers/Resolver.scala

```diff
@@ -49,39 +54,31 @@ object Resolver {
   final case class InProjectResolver(
       id: Iri,
       project: ProjectRef,
-      priority: Priority,
+      value: InProjectValue,
       source: Json,
       tags: Map[Label, Long]
-  ) extends Resolver
+  ) extends Resolver {
+    override def priority: Priority = value.priority
+  }
 
   /**
     * A resolver that can look across several projects.
     */
   final case class CrossProjectResolver(
       id: Iri,
       project: ProjectRef,
-      resourceTypes: Set[Iri],
-      projects: NonEmptyList[ProjectRef],
-      identityResolution: IdentityResolution,
-      priority: Priority,
+      value: ResolverValue,
       source: Json,
       tags: Map[Label, Long]
-  ) extends Resolver
+  ) extends Resolver {
+    override def priority: Priority = value.priority
+  }
 
   val context: ContextValue = ContextValue(contexts.resolvers)
 
   implicit val resolverEncoder: Encoder.AsObject[Resolver] = {
```

Comment:
shouldn't it be `private` or `private[resolver]` ? Usually we want to expose only the `JsonLdEncoder[Resolver]` and not the `[Encoder.AsObject[Resolver]]`